extensions/ao-raycast/scripts/convert-svg.ts

```diff
@@ -0,0 +1,25 @@
+import sharp from 'sharp';
+import path from 'path';
+
+async function convertAoLogo() {
+    const inputPath = path.join(process.cwd(), 'assets', 'ao.svg');
+    const outputPath = path.join(process.cwd(), 'assets', 'ao.png');
```

Comment:
You could check in [environment](https://developers.raycast.com/api-reference/environment) instead